plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -251,10 +303,26 @@ private static boolean isNewStringFromByteArrayParams(PsiExpression[] args) {
       return false;
     }
 
-    private ProblemDescriptor getRedundantCaseEqualsProblem(PsiMethodCallExpression call) {
-
+    private ProblemDescriptor getStripAndBlankProblem(PsiMethodCallExpression call) {
       PsiExpression equalTo = PsiUtil.skipParenthesizedExprDown(call.getArgumentList().getExpressions()[0]);
+      if (equalTo == null) return null;
```

Comment:
Same here: we already have `Java | Performance | 'String.equals()' can be replaced with 'String.isEmpty()'`, so `.equals("")` will be replaced with `isEmpty()` by another inspection. No need to support both steps at once.

Reply:
Removed